features/redesigned_searches.feature

```diff
@@ -226,31 +204,32 @@ Feature: Search - redesign
       | medline_title | medline_tid | url                                                                          |
       | Hippo1        | 24680       | https://www.nlm.nih.gov/medlineplus/Hippopotomonstrosesquippedaliophobia.html |
     When I am on english-nih's search page
-    And I fill in "query" with "hippopotomonstrosesquippedaliophobia"
-    And I press "Search" within the search box
-    Then I should see "Hippopotomonstrosesquippedaliophobia and Other Irrational Fears" within the med topic govbox
+    And I fill in "searchQuery" with "hippopotomonstrosesquippedaliophobia"
+    And I press "Search"
+    Then I should see "Hippopotomonstrosesquippedaliophobia and Other Irrational Fears"
```

Comment:
@yogesh27 By removing the "within the med topic govbox" here we can't be confident that either this (or the spanish result below) are coming from med topics rather than just a standard search.  We need to be sure these tests are testing medline...

Reply:
@lorawoodford This is fixed.